docs/examples/en/exporters/DRACOExporter.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<p class="desc">
+			An exporter to compress geometry with the Draco library. <br /><br />
+			[link:https://google.github.io/draco/ Draco] is an open source library for compressing and
+			decompressing 3D meshes and point clouds. Compressed geometry can be significantly smaller,
+			at the cost of additional decoding time on the client device.
+		</p>
+
+		<p>
+			Standalone Draco files have a `.drc` extension, and contain vertex positions,
+			normals, colors, and other attributes. Draco files *do not* contain materials,
+			textures, animation, or node hierarchies – to use these features, embed Draco geometry
+			inside of a glTF file. A normal glTF file can be converted to a Draco-compressed glTF file
+			using [link:https://github.com/AnalyticalGraphicsInc/gltf-pipeline glTF-Pipeline].
+		</p>
```

Comment:
This section makes sense for `DRACOLoader` but not necessarily for the exporter. Users don't need `glTF-Pipeline` if they export to drc via `DRACOExporter`.

Reply:
Need more advice on this one:

That makes sense, happy to ditch the paragraph. I'm worried about a misconception that made me leave it in.

This exporter won't help with updating ".glTF" files to contain draco encoded geometry buffers directly.  That's what everyone wants to do.  Like you said, this exporter only produces ".drc" files.  To draco encode parts of a glTF, they would still need the pipeline.  Draco compressing a blob of data within a glTF, setting the extention attributes, embedding the correct meta, that's far beyond what this thing can do.

Reply:
You're right. After reading the paragraph again, it might be good to leave it in to avoid DRACO compression in context of glTF.